libs/shell/feature/src/lib/shell-routing.module.ts

```diff
@@ -199,6 +199,16 @@ const routes: Routes = [
       ),
     canLoad: [AuthGuard],
   },
+  {
+    path: 'job-summary-parent-view',
```

Comment:
Please add the auth guards so the routes arent accessible by non logged in users,